hoot-core/src/main/cpp/hoot/core/util/GeometryUtils.cpp

```diff
@@ -173,10 +173,10 @@ QString GeometryUtils::toString(const Envelope& e)
 {
   const int precision = ConfigOptions().getWriterPrecision();
   return QString("%1,%2,%3,%4").
-      arg(e.getMinX(), 0, 'g', precision).
-      arg(e.getMaxX(), 0, 'g', precision).
-      arg(e.getMinY(), 0, 'g', precision).
-      arg(e.getMaxY(), 0, 'g', precision);
+      arg(e.getMinX(), 0, 'f', precision).
+      arg(e.getMaxX(), 0, 'f', precision).
+      arg(e.getMinY(), 0, 'f', precision).
+      arg(e.getMaxY(), 0, 'f', precision);
```

Comment:
Changed the representation from `g` to `f` because `g` means `use e or f format, whichever is the most concise` where `e` is scientific notation i.e. `[-]9.9e[+|-]999` which we definitely don't want.
http://doc.qt.io/archives/qt-4.8/qstring.html#argument-formats